Dockerfile

```diff
@@ -35,6 +35,10 @@ RUN ["chmod", "-R", "u=rwX,go=rX", "/rootfs"]
 
 FROM scratch
 
+LABEL org.opencontainers.image.documentation='https://icinga.com/docs/icinga-db/latest/doc/01-About/' \
+      org.opencontainers.image.source='https://github.com/Icinga/icingadb' \
+      org.opencontainers.image.licenses='GPL-2.0+'
+
```

Comment:
Would you consider also adding `.vendor`, holding "Icinga GmbH" as its value, and `.title`?

Reply:
Sure why not. There are a few others which could be added as well.